helm/prometheus-kafka-adapter/templates/service.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "prometheus-kafka-adapter.fullname" . }}
+  name: {{ tpl .Values.service.name . }}
```

we don't need to support tpl

---

Isn't that the only way to allow for variables in the value? In this case keeping the default the same, set to `"prometheus-kafka-adapter.fullname"`

---

```suggestion
  name: {{ include "prometheus-kafka-adapter.fullname" . }} 
```
This is the way to use fullname as the value for the service name 😄